<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimulationAnalysis</class>
 <widget class="QDialog" name="SimulationAnalysis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="LE_datFile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>401</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Simulation output file:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_openscript">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>140</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>open DNA Analysis script</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_runAnal">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>280</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Run Analysis</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_finish">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Finish</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
